client-app/shared/layout/components/header/_internal/mobile-header.vue

```diff
@@ -1,23 +1,39 @@
 <template>
-  <div class="fixed z-40 w-full shadow-md">
-    <div class="relative z-10 h-14 flex justify-between items-center bg-white">
-      <router-link to="/" class="px-6">
-        <VcImage src="/static/images/common/logo.svg" class="h-9" lazy />
-      </router-link>
-
-      <div class="flex items-center h-full pr-3">
-        <button v-show="!searchBarVisible" class="h-full px-3" @click="showSearchBar">
-          <i class="fas fa-search text-lg text-[color:var(--color-primary)]" />
-        </button>
-
-        <button class="h-full px-3" @click="mobileMenuVisible = true">
-          <i class="fas fa-bars text-2xl text-[color:var(--color-primary)]" />
-        </button>
+  <div ref="headerElement" class="fixed z-40 w-full shadow-md bg-[color:var(--color-header-bottom-bg)]">
+    <!-- region Default slot -->
+    <transition :name="isAnimated ? 'slide-fade-top' : ''" mode="out-in">
+      <div v-if="customSlots.default">
+        <component :is="customSlots.default" />
       </div>
-    </div>
 
-    <!-- Mobile Search Bar -->
-    <div v-show="searchBarVisible" class="flex p-4 bg-gray-800 select-none">
+      <div v-else class="relative w-full z-10 h-14 flex justify-between items-center">
+        <!-- region Left slot -->
+        <component v-if="customSlots.left" :is="customSlots.left" />
+
+        <router-link v-else to="/" class="px-6">
+          <VcImage src="/static/images/common/logo.svg" class="h-9" lazy />
```

Comment:
Maybe we shouldn't use `lazy` here - logo sometimes appears later and search bar change its width.